src/asciidoctor-core-api.js

```diff
@@ -948,15 +958,45 @@ Title.$$proto.hasSubtitle = function () {
 
 // Inline API
 
-/** @namespace */
+/**
+ * @namespace
+ * @extends AbstractNode
+ */
 var Inline = Opal.Asciidoctor.Inline;
 
 /**
+ * Get the converted String text of this Inline node, if applicable.
+ *
  * @memberof Inline
- * @extends AbstractNode
+ * @returns {string} - returns the converted String text for this Inline node, or undefined if not applicable for this node.
  */
-Inline.$$proto.convert = function () {
```

Comment:
Oops, I was moving lines around and took away too many.

The reason I had initially deleted it was because I thought AbstractNode provide convert. As it turns out, it does not, though it probably should. It's something I've noted down to look at in core.